libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/changeSharingHandler.test.ts

```diff
@@ -21,7 +21,7 @@ describe("change dashboard sharing handler", () => {
                 "GDC.DASH/EVT.SHARING.CHANGED",
             );
 
-            expect(event.payload.newShareStatus).toEqual("public");
+            expect(event.payload.newShareProps).toEqual("public");
```

Comment:
This is fishy, now the `event.payload.newShareProps` is an object, this expect should fail, right?

Reply:
may be it will, I missed to run tests after my changes locally :(